src/widgets/DiagramWidget.tsx

```diff
@@ -47,6 +48,7 @@ export class DiagramWidget extends React.Component<DiagramProps, DiagramState> {
 		allowCanvasTranslation: true,
 		allowCanvasZoom: true,
 		inverseZoom: false,
+		maxNumberPointsPerLink: 3, // a sane default
```

Comment:
Again, most of the changes are tslinting.

This is the new property with a (in my opinion) sane default.  
Open to changes though.

Reply:
Either this needs to be set to null (which is a backwards compatible sane default) or we need to have a way to intercept the events with a preventDefault(). If we did that, we could control whether all events are allowed to continue firing. The reason I'm stating this, is because the proposed solution is more universal in design, but having said that, I'm happy to include this until we can achieve this.

Reply:
Right, so instead of `null` - which means we would have to introduce a special handling for it further down the code - I've set the default to `Infinity`, which effectively translates into the current behaviour and we can still perform the number comparison without any special treatment. :)